selfdrive/ui/qt/maps/map.cc

```diff
@@ -460,27 +482,14 @@ void MapInstructions::updateInstructions(cereal::NavInstruction::Reader instruct
   QString type = QString::fromStdString(instruction.getManeuverType());
   QString modifier = QString::fromStdString(instruction.getManeuverModifier());
   if (!type.isEmpty()) {
-    QString fn = "../assets/navigation/direction_" + type;
+    QString fn = "direction_" + type;
     if (!modifier.isEmpty()) {
       fn += "_" + modifier;
     }
     fn += ICON_SUFFIX;
     fn = fn.replace(' ', '_');
-
-    // for rhd, reflect direction and then flip
-    if (is_rhd) {
-      if (fn.contains("left")) {
-        fn.replace("left", "right");
-      } else if (fn.contains("right")) {
-        fn.replace("right", "left");
-      }
-    }
-
-    QPixmap pix(fn);
-    if (is_rhd) {
-      pix = pix.transformed(QTransform().scale(-1, 1));
-    }
-    icon_01->setPixmap(pix.scaledToWidth(200, Qt::SmoothTransformation));
+    bool rhd = is_rhd && fn.contains("_left") || fn.contains("_right");
```

Comment:
can you add some parentheses here to make this less ambiguous?

Reply:
added.this is also a bug if without parentheses

Reply:
is there a compiler flag we can set to make the build fail?

Reply:
Not sure, hope there is such a flag. i'll look up the documentation and add it if there is.